users/faq-parts/usage.rst

```diff
@@ -87,6 +87,17 @@ block) from A, and then as A gets rescanned remove the files from A.
 A workaround would be to copy first from A to B, rescan B, wait for B to
 rebuild on remote ends, and then delete from A.
 
+Can I help initial sync by copying files manually?
+--------------------------------------------------
+
+If you have a large folder that you want to keep in sync over a not-so-fast network, and you have the possibility to move all files to the remote site in a faster manner, here is a procedure to follow:
+
+- Create the folder on site 1, but don't share it with anyone.
```

Comment:
```suggestion
- Create the folder on device A, but don't share it with device B yet.
```

Reply:
I don't think we use "sites" to refer to "devices" anywhere, so I'd suggest to stick to "device" instead.